### ISSUE TITLE
"last_release" URLs

### ISSUE THREAD

**Issue by ceceliadid** [ufs-community/ufs-mrweather-app#68]
@arunchawla-NOAA @rsdunlapiv @ligiabernardet @climbfuji @junwang-noaa It would be tremendously useful to define URLS for the "last_release" of code and documentation for the app, the weather model, and utilities like nceplibs, as soon as possible. The definition would get updated with each new release. This was incredibly useful to ESMF over the years from a communicatoin point of view because it meant that you could create websites and documents that were far easier to maintain, if the specific version # for the reference did not matter that much.  It would be very helpful to have this ready for pre-release graduate student testing.

**Comment by arunchawla-NOAA** [comment]
How do we do this till we have the first release?
---------------------------------------------------------------
Arun Chawla
Chief
Engineering & Implementation Branch
Room 2083
National Center for Weather & Climate Prediction
5830 University Research Court
College Park, MD 20740
Phone : 301-683-3740
Mobile : 240-564-5675
     Fax : 301-683-3703
------------------------------------------------------------

**Comment by ceceliadid** [comment]
I think you need to have something that looks like a candidate tag for the weather model, nceplibs, and the associated WM UG documentation by the end of this month. We need these to do to the weather model GST. Create the last_release structure and for now point it to those candidate tags. When you have the actual release, you update the last_release tag with the new values.

**Comment by arunchawla-NOAA** [comment]
ok so if we create beta snapshot tags and then put them in the last release
structure then people will think this is an actual release and start using.
I am not sure if that is a smart idea. I have no problem setting up beta
tags, unless others have concerns.
---------------------------------------------------------------
Arun Chawla
Chief
Engineering & Implementation Branch
Room 2083
National Center for Weather & Climate Prediction
5830 University Research Court
College Park, MD 20740
Phone : 301-683-3740
Mobile : 240-564-5675
     Fax : 301-683-3703
------------------------------------------------------------

**Comment by ceceliadid** [comment]
Well, you already have a branch called ufs_public_release so I think if people were going to get confused by what you name things in the repo they would have already.
The model and app wikis say "this repository is not yet ready for general use" and I think about a billion people have been told that the release is upcoming but not out yet. It really seems ok to do this to me and a month before the release the right timeframe to get it ready. That way we can use it for the documents and web pages we need to create.

**Comment by junwang-noaa** [comment]
For previous FV3GFS public release, we did have a beta version coming out
and connected both internal and external developers outside release team to
run the beta version. I remember we got some useful feedback and we put
them in the final release tag. So I think we can target some users to try
the beta version, but it's better not to announce the beta version to the
public as it may cause confusion.

Jun

**Comment by ceceliadid** [comment]
@junwang-noaa I agree that the beta tags are just for internal and GST testing, not for public announcement. The public mostly doesn't need to think about the last_release structure, that is just a huge convenience for document and website maintenance.

**Comment by ligiabernardet** [comment]
The release codes (beta versions and actual release versions) should be tagged. We can use use ReadTheDocs to display multiple releases, latest release, and top of development (auto-updating). You can see a prototype of that in the ReadTheDocs for the CCPP Technical documentation. On the lower right there is a button to choose the various releases (https://ccpp-techdoc.readthedocs.io/en/latest/).

**Comment by ceceliadid** [comment]
That looks perfect.

**Comment by ceceliadid** [comment]
@arunchawla-NOAA Is there a plan for when these latest_release tags would be going on the code and documentation?

**Comment by arunchawla-NOAA** [comment]
@ceceliadid how did we want these tags to look? And should we just tag the weather app as the model is called from it? Are the library builds mentioned there?

**Comment by ceceliadid** [comment]
@arunchawla-NOAA @llpcarson @ligiabernardet I would suggest seeing if v1.0.0 tags are going to be moved in the UGs for anything else and deal w/this on Monday. I see places where it would be useful to have this, but at this point it's probably better to wait for the first patch releases. I don't want to change the wikis or Portal for this now. The proposal was ufs-latest-release or latest-release only on the WM, app, and NCEPLIBS/NCEPLIBS-external.

**Comment by ligiabernardet** [comment]
Should we close this issue for now then?

**Comment by ceceliadid** [comment]
My preference would be to keep it open so it can be done before it gets forgotten. I put a post release tag on it - it seems like a good time to start identifying some of those issues.  It should be pretty quick and easy.
